### PR TITLE
docs(asset-tree): rewrite stories in new format

### PR DESCRIPTION
**Pull request by amevck**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-539.gearboxjs.preview.cogniteapp.com

File: src/components/AssetTree/stories/basic.story.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+This type describes the parameters the `onSelect` function is called with.
+The type can be imported from @cognite/gearbox:
+
+```jsx
+import { OnSelectAssetTreeParams } from '@cognite/gearbox';
+```
+
+#### Asset
+
+The Asset type documentation can be found at https://js-sdk-docs.cogniteapp.com/interfaces/asset.html
```
Comment (polomani) [inline src/components/AssetTree/stories/basic.story.mdx]: that's an outdated link, here is a proper one
```suggestion
The Asset type documentation can be found at https://cognitedata.github.io/cognite-sdk-js/classes/asset.html
```

File: src/components/AssetTree/stories/examples.story.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  },
+};
+```
+<Preview withSource='open'>
+  <Story name="With Theme">  
+    <ThemeProvider theme={exampleTheme}><AssetTree /></ThemeProvider>
+  </Story>
+</Preview>
+
+## With Asset Id
```
Comment (polomani) [inline src/components/AssetTree/stories/examples.story.mdx]:
```suggestion
## With custom root assets
```

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-539.gearboxjs.preview.cogniteapp.com

File: src/components/AssetTree/stories/examples.story.mdx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  },
+};
+```
+<Preview>
+  <Story name="With Theme">  
+    <ThemeProvider theme={exampleTheme}><AssetTree /></ThemeProvider>
+  </Story>
+</Preview>
+
+## custom root assets
```
Comment (polomani) [inline src/components/AssetTree/stories/examples.story.mdx]: you use uppercase first letter in other places

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-539.gearboxjs.preview.cogniteapp.com

File: src/components/AssetTree/stories/basic.story.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+<Preview>
+  <Story name="Basic Usage">
+    <AssetTree/>
+  </Story>
+</Preview>
+
+## Available props
+
+<Props of={AssetTree} />
```
Comment (mrtraser) [inline src/components/AssetTree/stories/basic.story.mdx]: Please, redo this in another way. It's possible to show props properly for this component

Comment (amevck) [reply src/components/AssetTree/stories/basic.story.mdx]: @mrtraser What did you mean by _another way_? edit the assetTree component?

Comment (mrtraser) [reply src/components/AssetTree/stories/basic.story.mdx]: check `AssetBreadcrumb` example

File: src/components/AssetTree/stories/basic.story.mdx [hunk 5]
```diff
@@ -0,0 +1,63 @@
+import { Meta, Story, Preview, Props, Source } from '@storybook/addon-docs/blocks';
```
Comment (mrtraser) [inline src/components/AssetTree/stories/basic.story.mdx]: Do we need to import `Source` here?

File: src/components/AssetTree/stories/examples.story.mdx [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+  list: {
+    fontFamily: 'Courier New',
+    fontSize: 'large',
+  },
+};
+```
+      
+<Preview>
+  <Story name="Custom Styles">  
+  <AssetTree
```
Comment (mrtraser) [inline src/components/AssetTree/stories/examples.story.mdx]: alignment

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-539.gearboxjs.preview.cogniteapp.com

**Comment by amevck** [comment]
Storybook `Props` component seems to have a bug with showing properties of class components. It works with functional components. 

In the `mdx` file

```jsx
<Props of={AssetTree} />
```
And use a class component like below, the Props aren't picked up:

```typescript
export interface AssetTreeProps {
  // ...
}
export class AssetTree extends React.Component<AssetTreeProps, AssetTreeState> {
  // ...
}
```

But props defined in `defaultProps` static field gets picked up. When I remove `defaultProps` no Props are picked up.

**Comment by polomani** [comment]
@amevck here is a fix for your problem
https://cognite-teams.slack.com/archives/GPUCM2QKD/p1576511565005000

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-539.gearboxjs.preview.cogniteapp.com

**Comment by amevck** [comment]
When we are using the imported `AssetTreeProps` interface like below:

```typescript
import { AssetTreeProps } from '../../interfaces';
```
The jsDocs doesn't get picked up.

But if the  AssetTreeProps interface is in the same file' then jsDocs are visible.

Could it be a bug in Storybook Addon Docs?

**Pull request merged**